utils/tools.js

```diff
@@ -1,6 +1,14 @@
 const forge = require('node-forge');
 const crypto = require('crypto');
 
+const d64 = data => forge.util.decode64(data);
```

Comment:
Avoid unnecessary blank lines.

Reply:
These const variables are representing as the function, should we remove the blank line before these functions? 🤔

Reply:
I think so.

Reply:
Unnecessary blank lines are removed. 👍